Formula/gnupg.rb

```diff
@@ -13,70 +13,92 @@ class Gnupg < Formula
     sha256 "f1521ac3df05904c32bdd52173f44d129e565edabdd935515519a087498575c4" => :mavericks
   end
 
-  depends_on "curl" if MacOS.version <= :mavericks
-  depends_on "libusb-compat" => :optional
+  option "with-gpgsplit", "Additionally install the gpgsplit utility"
+  option "without-libusb", "Disable the internal CCID driver"
+
+  deprecated_option "without-libusb-compat" => "without-libusb"
+
+  depends_on "pkg-config" => :build
+  depends_on "sqlite" => :build if MacOS.version == :mavericks
+  depends_on "npth"
+  depends_on "gnutls"
+  depends_on "libgpg-error"
+  depends_on "libgcrypt"
+  depends_on "libksba"
+  depends_on "libassuan"
+  depends_on "pinentry"
+  depends_on "gettext"
+  depends_on "adns"
+  depends_on "libusb" => :recommended
+  depends_on "readline" => :optional
+  depends_on "homebrew/fuse/encfs" => :optional
+
+  # ssh-import.scm fails during "make check" for sandboxed builds
+  # Reported 1 Mar 2017 https://bugs.gnupg.org/gnupg/issue2980
+  patch :DATA
 
   def install
     args = %W[
       --disable-dependency-tracking
       --disable-silent-rules
       --prefix=#{prefix}
-      --disable-asm
-      --program-suffix=1
+      --sbindir=#{bin}
+      --sysconfdir=#{etc}
+      --enable-symcryptrun
+      --with-pinentry-pgm=#{Formula["pinentry"].opt_bin}/pinentry
     ]
-    args << "--with-libusb=no" if build.without? "libusb-compat"
+
+    args << "--disable-ccid-driver" if build.without? "libusb"
+    args << "--with-readline=#{Formula["readline"].opt_prefix}" if build.with? "readline"
 
     system "./configure", *args
-    system "make"
-    system "make", "check"
 
-    # we need to create these directories because the install target has the
-    # dependency order wrong
-    [bin, libexec/"gnupg"].each(&:mkpath)
     system "make", "install"
```

Comment:
Can you copy the logic from the existing `gnupg2` to install vanilla symlinks to `gpg` and `gpgv`.